build.gradle

```diff
@@ -46,7 +46,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.0'
+        classpath 'com.android.tools.build:gradle:4.1.1'
```

Comment:
I have not updated AS to v4.1.1 yet :( Maybe you can create a new PR for this update and update the library dependency too.